frontends/python/main.py

```
@@ -0,0 +1,393 @@
+) -> int:
+    """Runs entire fuzz pass"""
+    if package is None:
+        package = resolve_package(fuzzer)
+        if package is None:
+            logger.error("No package. Exiting early now as the results will not be good")
+            package = ""
+
+    # If indicated, scan package directory recursively to identify .py files.
+    scanned_sources = []
+    if scan:
+        for root, dirs, files in os.walk(package):
```

Ruff (0.8.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)